applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -136,29 +110,11 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	struct ble_iso_data *iso_received = NULL;
 
 #if (CONFIG_AUDIO_DEV == GATEWAY)
-	static uint32_t packet_count_r;
-
-	switch (channel_index) {
-	case AUDIO_CH_L:
-		/* Proceed */
-		break;
-	case AUDIO_CH_R:
-		packet_count_r++;
-		if ((packet_count_r % 1000) == 0) {
-			LOG_DBG("Packets received from right channel: %d", packet_count_r);
-		}
-
-		return;
-	default:
-		LOG_ERR("Channel index not supported");
+	if (channel_index != AUDIO_CH_L) {
+		/* Only left channel RX data in use on gateway */
 		return;
 	}
-#endif /* (CONFIG_AUDIO_DEV == GATEWAY) */
-
-#if (CONFIG_BLE_ISO_TEST_PATTERN)
-	ble_test_pattern_receive(p_data, data_size, bad_frame);
-	return;
-#endif /* (CONFIG_BLE_ISO_TEST_PATTERN) */
+#endif
```

Comment:
Still missing the /* (CONFIG_AUDIO_DEV == GATEWAY) */ comment

Reply:
Resolved